test/urls.test.js

```diff
@@ -16,7 +17,7 @@ describe('urls', () => {
 
   describe('getApiServerUrl()', () => {
     afterAll(() => {
```

Comment:
Not changes here, but `afterAll` should be `afterEach`

Reply:
In fact, this `afterAll` can be omitted and the setup methods in the parent scope should be `*Each`s